server/spec/middlewares/websocket_backend_spec.rb

```diff
@@ -4,10 +4,23 @@
   let(:app) { spy(:app) }
   let(:subject) { described_class.new(app) }
 
+  before(:all) do
+    EM.run
```

Comment:
`EM.run` blocks the calling thread: http://www.rubydoc.info/github/eventmachine/eventmachine/EventMachine#run-class_method